DotNetHowToSynonyms/DotNetHowToSynonyms/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("AzureSearch.DotNetHowTo")]
```

Comment:
You should change the assembly info too so it's not the same as in the DotNetHowTo project.